src/main/java/frc/robot/OI.java

```diff
@@ -8,16 +8,21 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.buttons.Button;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
+import frc.robot.commandgroups.LaunchActuator;
 
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
-  Joystick logitech = new Joystick(0);
+  Joystick logitech = new Joystick(RobotMap.joystickNumber);
   Double deadzone = 0.05;
+  int actuatorButtonNumber=1;
```

Comment:
move the constant values to the top of the file all declared as final. I should be able to look at the top of a class and see all the static final variables so that I can quickly change them if needed at the competition.